modules/rosa-creating-cluster.adoc

```diff
@@ -17,11 +17,13 @@ You have completed the installation prerequisites.
 
 Enter the following command to create your cluster with the default cluster settings.
 
+* To view other options when creating a cluster, enter `rosa create cluster --help`.
+
+* To follow a set of interactive prompts, enter `rosa create cluster --interactive`.
++
 [NOTE]
 ====
-To view other options when creating a cluster, enter `rosa create cluster --help`.
-
-To follow a set of interactive prompts, enter `rosa create cluster --interactive`.
+Multi-availability zones (AZ) are recommended for ROSA production workloads. The default is a single availability zone. Use either `--help` to set the option manually or use interactive mode to be prompted for this setting.
```

Comment:
I think that "either" is throwing me off on the second sentence. I think you could just say "Use `--help` to set the option manually or use interactive mode to be prompted for this setting."

Also, I can't imagine that the `--help` flag actually sets the option manually. I would think that help might just give you an example of how to set the option manually. Should it be "Use `--help` for an example of how to set the option manually or use interactive mode to be prompted for this setting." ?

Reply:
Agreed to wqhat @bergerhoffer said.  Also s/Multi-availability/Multiple availability zones

and s/ROSA//